src/frontend/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN npm run build
 
 FROM node:18-alpine AS runner
 WORKDIR /app
-RUN apk add --no-cache protoc
+RUN apk add --no-cache protobuf-dev protoc
```

Comment:
I think this change is duplicated in #631, #632 #633. I'm not sure, is it necessary to add this change to every pr?